### DISCUSSION TITLE
A guide to adding subscriptions to a Quarto site

### DISCUSSION THREAD

**Discussion by ForBo7** [quarto-dev/quarto-cli#3746] [Show and Tell]
Hello!

@Isaac-Flath and I worked together on creating a guide on how to add email subscriptions to a Quarto site. This guide should help to fill in the blanks of the Quarto docs.

You can read the guide [here](https://forbo7.github.io/forblog/posts/7_blog_subscriptions.html).

Let us know about any feedback, and any corrections and improvements!

**Comment by dragonstyle** [comment]
Really cool thank you for sharing this!

**Comment by HangenYuu** [comment]
Hi,

I followed along the guide and added a subscription to my site. It works on the Home/About pages, but crashes on blog posts.
Here are the details:

subscribe.html and option added in _quarto.yml (I use MailerLite):
![image](https://user-images.githubusercontent.com/97870091/216824506-9b139a01-e7f9-46b4-a6cb-6758282e8c70.png)
![image](https://user-images.githubusercontent.com/97870091/216824527-3a2af4d1-1c44-4a52-81f1-31fc340cd001.png)

Result on homepage:
![image](https://user-images.githubusercontent.com/97870091/216824558-0a687d8e-2731-4c2c-a4b9-b34fead2ab3a.png)

Result on blog page:
![image](https://user-images.githubusercontent.com/97870091/216824588-9c5553dc-1bab-436a-9f13-e4845c6aebf8.png)

The source code HTML for homepage (correct):
![image](https://user-images.githubusercontent.com/97870091/216824624-45606901-98fc-46f2-8618-a3144bba3477.png)

The source code HTML for blog page (lmao):
![image](https://user-images.githubusercontent.com/97870091/216824676-9a18835f-882f-4080-9e2a-a27e8aa41884.png)

Do you have any ideas how this happened/how to fix this? The Quarto docs and the source code for [@Isaac-Flath](https://github.com/Isaac-Flath)'s website did not yield anything. Thank you.

**Reply by dragonstyle** [reply]
I believe this is a bug that is fixed in later versions of Quarto - can you check what version of Quarto you're using (and update to our latest stable release if you aren't already using that one)?

**Reply by HangenYuu** [reply]
I was using Quarto 1.1.xxx. After updating to the latest (1.2.xxx), the issue was fixed. Thanks.

(I do not remember what was xxx, but does not matter.)
